index.html

```diff
@@ -3598,13 +3587,13 @@ <h3>Evidence</h3>
 
         <p class="note">
 The `evidence` [=property=] provides different and complementary
```

Comment:
This is the first part of one suggestion --
```suggestion
The `evidence` [=property=] provides information that is different from and
```